src/AbstractTwigCallable.php

```diff
@@ -79,6 +79,9 @@ public function getDynamicName(): string
         return $this->dynamicName;
     }
 
+    /**
+     * @return callable
```

Comment:
This is wrong. The callable of a AbstractTwigCallable might not be a PHP callable, due to the case of Twig runtimes.

See the constructor parameter type (on child classes, as it is missing on the constructor of that class currently)